action_server/src/action_server/actions/answer_question.py

```diff
@@ -50,6 +50,7 @@ def _parse_context(context_dict):
     def _configure(self, robot, config):
         self._robot = robot
 
+        #To do: remove this dependency!
```

Comment:
@LarsJanssenTUe can you make an issue to do this? SPR is no longer used. 
If the knowledge is used in multiple places, maybe it should be common knowledge

Reply:
The knowledge is only used in this action and parts of the actual SPR challenge. Should the questions then still be added to the common knowledge? Or should I make a separate question knowledge file?

Reply:
Ping @LoyVanBeek

Reply:
If it's ahared between several challenges, I say it's common. And putting it in a separate file is OK, when that's imported into `common` or something.

Reply:
#114